t/unit/app/test_app.py

```diff
@@ -696,6 +696,13 @@ class Config:
             assert exc.args[0].startswith('task_default_delivery_mode')
             assert 'CELERY_DEFAULT_DELIVERY_MODE' in exc.args[0]
 
+    def test_config_form_object__module_attr_does_not_exist(self):
+        with pytest.raises(ModuleNotFoundError) as exc:
+            self.app.config_from_object(f'{__name__}.bar')
+            # the module must exist, but it should not have the config attr
+            assert self.app.conf.broker_url is None
```

Comment:
Shouldn't this and the next assertion be outside the `with` scope?

Reply:
is this resolved? @frolenkov-nikita